environment.yaml

```diff
@@ -6,11 +6,13 @@ dependencies:
 - click
 - cftime
 - fsspec=0.8.7 # Pinned req. for adlfs < 0.7
+- gcsfs
 - numpy
 - pip
 - pytest
 - python=3.8
 - rechunker
+- s3fs
```

Comment:
Super cool how platform-agnostic this is now!

Reply:
Yeah, it's platform-agnostic (as long as you're in S3, GCS, or AZ). I hope it works.